test/helper.sh

```diff
@@ -24,4 +24,4 @@ write_pacman_conf() {
 ignore() { yes | prompt_to_ignore "$@" >/dev/null; }
 
 # Load implementation
-source "$SRCDIR"/downgrade
+source "$SRCDIR"/downgrade.in
```

Comment:
I don't know that this will work, or if it does I don't think it would always. I think this needs to continue to use `downgrade`, and the test just needs to depend on the `downgrade` target.

Reply:
It works with `downgrade.in`, since it only needs to source the variables/functions. If you want to keep it as `downgrade`, then you would probably need to add the `config` phony as a dependency to the `test` phony.